### ISSUE TITLE
Unable to calculate route (Index=155, size=0)

### ISSUE THREAD

**Issue by raindropsfromsky** [osmandapp/Osmand#6599]
In the last week, a few nightlies are having a peculiar problem: When I set a destination (even from history), OsmAnd reports:

Fast route calculation failed. 
Falling back to slow calculation  (Index=155, size=0).

And after some time it reports
Could not calculate the route
(Index=155, size=0)

Note that the GPS location is available at this time.
The last nightly to have this problem is dated 1st March.

**Comment by vshcherb** [comment]
We need some coordinates to reproduce it

**Comment by raindropsfromsky** [comment]
Unfortunately this is not repeatable any more: The same points from history are working now.

**Comment by jfburke3** [comment]
I run into this randomly, usually when I'm sitting in the car trying to set up navigation for a trip, but often have trouble reproducing it after I get to my destination.

**Comment by scaidermern** [comment]
Try exporting (saving, sharing) the route as soon as it happens. This should make it reproducible.

**Comment by raindropsfromsky** [comment]
Here are two locations that are having this problem:

Varthur Road, Bellanduru
Location: geo:12.957012,77.700745?z=12
https://osmand.net/go?lat=12.957012&lon=77.700745&z=12

Dodsworth Enclave
Location: geo:12.970769,77.74456?z=13
https://osmand.net/go?lat=12.970769&lon=77.74456&z=13

The first address had problem earlier also, but then it started working fine.
(I picked it out of history)

The second address is new. It was working fine yesterday, but I installed the latest nightly just now (7th March 2019), and both addresses are having the same problem.
